maintaining.rst

```diff
@@ -0,0 +1,276 @@
+Collection maintainers backport merged pull requests to stable branches
+following the `semantic versioning <https://semver.org/>`_ and release policies of the collections.
+
+For more information about the process, refer to the `Backporting guidelines <https://docs.ansible.com/ansible/devel/community/development_process.html#backporting-merged-prs-in-ansible-core>`_.
+
+For convenience, backporting can be implemented automatically using GitHub bots (for example, with the `Patchback app <https://github.com/apps/patchback>`_) and labeling like it is done in `community.general <https://github.com/ansible-collections/community.general>`_ and `community.network <https://github.com/ansible-collections/community.network>`_.
+
+.. _Releasing:
+
+Releasing
+=========
+
+Collection maintainers release all supported stable versions of the collections regularly,
+provided that there have been enough changes merged to release.
+
+Generally, releasing in the collections consists of:
+
+  1. Planning and announcement.
+  2. Generating a changelog.
+  3. Creating a release git tag and pushing it.
+  4. Automatic publishing the release tarball on `Ansible Galaxy <https://galaxy.ansible.com/>`_ by Zuul.
+  5. Final announcement.
+
+For more information about the process, refer to the `Releasing guidelines <releasing.rst>`_.
+
+.. _Expanding community:
+
+Expanding community
+===================
+
+Increasing a number of active contributors and maintainers
+----------------------------------------------------------
+
+Maintainers are interested in increasing a number of active long-term contributors for a collection they maintain.
+
+Contributors are reviewers, issue or pull request authors, testers, maintainer, and all other people who help develop the project.
+
+Every regular contributor once was a newcomer. Make the first experience as positive as possible to make the new people coming back.
+
+Good development documentation makes contributors life much easier. Get feedback from new contributors if there were things they struggled with when working on their proposals and improve the documentation correspondingly.
+
+Create the ``CONTRIBUTING`` file in your repository. In there, add a link to the `Quick-start guide <create_pr_quick_start_guide.rst>` as well as to other guidelines describing things specific to your collection.
+
+Make contributors feel welcome. Greet and thank contributors impersonally in ``README`` and individually in their proposals.
+Thank all participants after merging or closing a proposal.
+
+Be responsive. Respond as quickly as possible. Even if you cannot review a proposal right now, greet and thank the author.
+
+Add and keep updated the ``CONTRIBUTORS`` file listing all the contributors including issue reporters and refer to it from your ``README``.
+
+Do not fix trivial non-critical bugs yourself. Instead, mentor a person who would like to contribute.
+Mark issues with labels like ``easy_fix``, ``help_wanted``, and ``documentation``.
```

Comment:
```suggestion
Mark issues with labels like ``easy_fix``, ``waiting_on_contributor``, and ``documentation``.
```

Reply:
I like the idea. I think they aren't fully interchangeable, maybe both?

Reply:
I haven't seen `help_wanted` tag on ansible/ansible or community repositories, also https://github.com/ansible-collections/community.general/pull/2602#discussion_r638678729.
Using one label for this type of issues would help both maintainers labeling and contributors finding these issues IMHO.

Reply:
persuaded:) done, thanks for explaining

Reply:
@aminvakil would you like to change `easy_fix` -> `easyfix` and `help_wanted` -> `waiting_on_contributor` in contributing.rst in this repo? Feel free to ping me, after raising a PR.

Reply:
I'd be glad to, I will in a couple of hours.